e2e/cypress/tests/integration/channel_settings/add_users_to_channel_spec.ts

```diff
@@ -87,14 +87,14 @@ describe('Channel Settings', () => {
         // # First add one user in order to see them disappearing from the list
         cy.get('#multiSelectList > div').first().then((el) => {
             const childNodes = Array.from(el[0].childNodes);
-            childNodes.map((child) => usernames.push(child.innerText));
+            childNodes.map((child: HTMLElement) => usernames.push(child.innerText));
 
             // # Get username from text for comparison
             username = usernames.toString().match(/@\w+/g)[0];
             cy.get('#multiSelectList').should('contain', username);
 
             // # Verify status wrapper is present within the modal list
-            cy.get(el).children(0).should('have.class', 'status-wrapper');
+            cy.get(el.toString()).children().first().should('have.class', 'status-wrapper');
```

Comment:
Revert this change

Reply:
I actually need to keep `.first()` as TS was complaining about `children(0)` is expecting a string.